src/views/subscribe/subscribe.js

```diff
@@ -0,0 +1,235 @@
+/** @jsx jsx */
+import { jsx, Styled } from 'theme-ui'
+import React from 'react'
+import "./style.css"
+import { Container } from '@theme-ui/components'
+import { useFormik } from 'formik';
+import ReactModal from 'react-modal'
+import { useState } from 'react'
+
+const Subscribe = () => {
+  const tag = `OpenEBS OF Newsletter`
+
+  const [showModal, setModal] = useState(false);
+  const [isSuccess, setIsSuccess] = useState(false);
+
+  const GATSBY_NEWSLETTER_BACKEND_URL =
+    process.env.GATSBY_NEWSLETTER_BACKEND_URL ? process.env.GATSBY_NEWSLETTER_BACKEND_URL : 'http://localhost:3000/mayadata-newsletter/openEBS-newsletter/';
+
+  const FormFields = {
+    firstName: '',
+    lastName: '',
+    email: '',
+  };
+
+  const Success = () => {
+    return (
+      <div
+        sx={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <Styled.h4 sx={{ color: 'white' }}>
+          You have successfully subscribed to OpenEBS Newsletter.
+        </Styled.h4>
+      </div>
+    )
+  }
+
+  const Failed = () => {
+    return (
+      <div
+        sx={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <Styled.h4 sx={{ color: 'white' }}>
+          Error in submitting the form, please try again!!!
+        </Styled.h4>
+      </div>
+    )
+  }
+
+  const sendData = (values, actions) => {
+    // TODO: Set up modular popup
+    if (values.email === '' || values.email === 'undefined') {
+      console.log(`please enter valid email`)
+    } else {
+      async function postData(url = '', data = {}) {
```

Comment:
`postData` function should be outside of `sendData` function. Please define it { `postData` } outside.

Reply:
and then call it in `sendData`.